.github/workflows/validate-pr-title.yml

```diff
@@ -0,0 +1,24 @@
+name: validate-pr-title
+on:
+  pull_request:
+    types: [edited, opened, reopened, synchronize]
+jobs:
+  title-matches-convention:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: 8BitJonny/gh-get-current-pr@1.4.0
+        id: PR
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          sha: ${{ github.event.pull_request.head.sha }}
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: regex-match
+        with:
+          text: ${{ steps.PR.outputs.pr_title }}
+          regex: '^(break|chore|docs|feat|fix|refactor|revert|style|test): .+$'
```

Comment:
are dependabots PRs respecting this convention or do they only start with `chore`?

Reply:
they only start with `chore: `, so they are respecting the convention